src/utils/url.ts

```diff
@@ -27,6 +32,44 @@ export const transactionIdAndSubmissionIdExistInRequest = (req: Request): boolea
 const getTransactionIdFromRequestParams = (req: Request): string => req.params[config.ROUTE_PARAM_TRANSACTION_ID];
 const getSubmissionIdFromRequestParams = (req: Request): string => req.params[config.ROUTE_PARAM_SUBMISSION_ID];
 
+/**
+ * This is required to extract the transactionId and submissionId prior to the request reaching its intended mount path
+ */
+export const getTransactionIdAndSubmissionIdFromOriginalUrl = (req: Request): TransactionIdAndSubmissionId | undefined => {
+  try {
+    logger.infoRequest(req, `extracting transactionId and submissionId from the req.originalUrl: ${req.originalUrl}`);
+    if (!req.originalUrl.includes('/transaction/') || !req.originalUrl.includes('/submission/')) {
```

Comment:
for the paths like '/submission/'  and 'submission' , '/transaction/' etc.. can they be added to the src/config/index.ts

Reply:
Agreed. This has been updated now.